modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -52,6 +52,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 
 
 
+== Bootstrap End-of-Life {centos} 6 Clients
+
+{centos} 6 is now at end-of-life, and the images provided in the client repository for this operating system are out of date.
+Bootstrapping new {centos} 6 clients using these packages will fail.
+This failure will not happen on {centos} 6 clients that are already installed and bootstrapped.
+
+If you need to bootstrap new {centos} 6 clients you can edit the existing repositories to reflect the correct URL.
+
+
+
+.Procedure: Troubleshooting Bootstrapping a New {centos} 6 Client
+. On the {centos} 6 client, at the command prompt, open the ``CentOS-Base.repo`` file, located in the `/etc/yum/repos.d/` directory.
+. Locate the ``baseurl`` line, pointing to ``mirrorlist.centos.org``.
+There may be more than one entry.
+For example:
++
+----
+baseurl=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os
+----
++
+. Edit each ``baseurl`` line to point to a ``vault.centos.org`` URL.
+For example:
++
+----
+baseurl=http://vault.centos.org/?release=7&arch=$basearch&repo=os
```

Comment:
Besides the problem, with `release=7`,  I don't think this is the right replacement to be done.

If I am not wrong the right URL is `https://vault.centos.org/centos/6/os/$basearch/`, but please **test** this on a real CentOS6 image in case I am wrong (a docker container should be enough as well).

Reply:
Right. @Loquacity vault is a direct URL and not the mirror list thingy.

The "old" url uses mirror list, while the new URL is just a plain URL to vault.
This is what I mean with "mapping" is needed. You need to turn the parameters of the mirror list into a real path.

Reply:
Right, that makes sense.